### PR TITLE
New cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[PYV Instrument Cal CSVs [DT-184]](https://trello.com/c/Eo546k32/184-pyv-instrument-cal-csvs-dt-184)

File: calibration/FLORTD/CGINS-FLORTD-01004__20220927.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+1004,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1004_Calibration_Files_2022-09-27.zip > BBFL2W-1004.dev
+1004,CC_dark_counts_cdom,47,
+1004,CC_dark_counts_chlorophyll_a,49,
+1004,CC_dark_counts_volume_scatter,51,
+1004,CC_depolarization_ratio,0.039,Constant
+1004,CC_measurement_wavelength,700,Constant
+1004,CC_scale_factor_cdom,8.386E-02,
+1004,CC_scale_factor_chlorophyll_a,1.084E-02,
+1004,CC_scale_factor_volume_scatter,1.726E-06,
+1004,CC_scattering_angle,124,Constant
```
Comment (asmith-whoi) [inline calibration/FLORTD/CGINS-FLORTD-01004__20220927.csv:11]: OK

File: calibration/FLORTD/CGINS-FLORTD-01217__20220926.csv [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+1217,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1217_Calibration_Files_2022-09-26.zip > BBFL2W-1217.dev
+1217,CC_dark_counts_cdom,49,
+1217,CC_dark_counts_chlorophyll_a,50,
+1217,CC_dark_counts_volume_scatter,47,
+1217,CC_depolarization_ratio,0.039,Constant
+1217,CC_measurement_wavelength,700,Constant
+1217,CC_scale_factor_cdom,8.812E-02,
+1217,CC_scale_factor_chlorophyll_a,1.155E-02,
+1217,CC_scale_factor_volume_scatter,1.826E-06,
+1217,CC_scattering_angle,124,Constant
```
Comment (asmith-whoi) [inline calibration/FLORTD/CGINS-FLORTD-01217__20220926.csv:11]: OK

File: calibration/FLORTD/CGINS-FLORTD-01299__20220913.csv [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+1299,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1299_Calibration_Files_2022-09-13.zip > BBFL2W-1299.dev
+1299,CC_dark_counts_cdom,51,
+1299,CC_dark_counts_chlorophyll_a,50,
+1299,CC_dark_counts_volume_scatter,51,
+1299,CC_depolarization_ratio,0.039,Constant
+1299,CC_measurement_wavelength,700,Constant
+1299,CC_scale_factor_cdom,9.139E-02,
+1299,CC_scale_factor_chlorophyll_a,1.208E-02,
+1299,CC_scale_factor_volume_scatter,1.837E-06,
+1299,CC_scattering_angle,124,Constant
```
Comment (asmith-whoi) [inline calibration/FLORTD/CGINS-FLORTD-01299__20220913.csv:11]: OK

**Review by asmith-whoi — APPROVED** [review]

File: calibration/FLORTD/CGINS-FLORTD-01004__20220927.csv (hunk 1 above)
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01004__20220927.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01116__20220923.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1116,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1116_Calibration_Files_2022-09-23.zip > BBFL2W-1116.dev > cal date taken from BBFL2W-1116 Characterization Sheet.pdf
+1116,CC_dark_counts_cdom,49,
+1116,CC_dark_counts_chlorophyll_a,49,
+1116,CC_dark_counts_volume_scatter,49,
+1116,CC_depolarization_ratio,0.039,Constant
+1116,CC_measurement_wavelength,700,Constant
+1116,CC_scale_factor_cdom,7.899E-02,
+1116,CC_scale_factor_chlorophyll_a,1.183E-02,
+1116,CC_scale_factor_volume_scatter,2.014E-06,
+1116,CC_scattering_angle,124,Constant
```
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01116__20220923.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01217__20220926.csv (hunk 2 above)
Comment (batryn) [reply calibration/FLORTD/CGINS-FLORTD-01217__20220926.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01299__20220913.csv (hunk 3 above)
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01299__20220913.csv:11]: Looks good

File: calibration/FLORTD/CGINS-FLORTD-01318__20220913.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+1318,CC_angular_resolution,1.076,Constant; source_file FLORT-D_BBFL2W_SN_1318_Calibration_Files_2022-09-13.zip > BBFL2W-1318.dev
+1318,CC_dark_counts_cdom,50,
+1318,CC_dark_counts_chlorophyll_a,52,
+1318,CC_dark_counts_volume_scatter,62,
+1318,CC_depolarization_ratio,0.039,Constant
+1318,CC_measurement_wavelength,700,Constant
+1318,CC_scale_factor_cdom,9.121E-02,
+1318,CC_scale_factor_chlorophyll_a,1.215E-02,
+1318,CC_scale_factor_volume_scatter,1.846E-06,
+1318,CC_scattering_angle,124,Constant
```
Comment (batryn) [inline calibration/FLORTD/CGINS-FLORTD-01318__20220913.csv:11]: Looks good

**Review by batryn — APPROVED** [review]
Looks good

**Pull request merged**